internal/server/handler.go

```diff
@@ -212,15 +213,35 @@ func (s *Server) SearchStatVar(
 // GetPropertyLabels implements API for Mixer.GetPropertyLabels.
 func (s *Server) GetPropertyLabels(
 	ctx context.Context, in *pb.GetPropertyLabelsRequest,
-) (*pb.GetPropertyLabelsResponse, error) {
-	return node.GetPropertyLabels(ctx, in, s.store)
+) (*pb.PayloadResponse, error) {
+	resp, err := node.GetPropertyLabels(ctx, in, s.store)
+	if err != nil {
+		return nil, err
+	}
+	jsonRaw, err := protojson.Marshal(resp)
+	// To make the API response backward compatible.
+	jsonRaw = jsonRaw[8 : len(jsonRaw)-1]
```

Comment:
Briefly explain in the comment what is the "8"?

Reply:
Done

Reply:
that is 6?  [len("data: {") : len(jssonRaw) - 1]

Reply:
Updated